### PR TITLE
TRAINING-16 Update Photo (to svg)

### PR DESCRIPTION
**Pull request by brucearctor**
SVG uploaded.  Changed site.xml to point to the svg.  It isn't clear what it does for sizing, and I still didn't dig into how to run locally for testing.  Seems that should be a soon/next step.

### CODE REVIEW

**Comment by brucearctor** [comment]
@lfrancke -- tagging you since you merged last attempt; a general point for all, given this is training.  We are lacking documentation on how to build the website, so that I can sensibly test/view locally.  My current approach is to try to change things and see how it looks, which isn't quite ideal for anyone.

**Comment by lfrancke** [comment]
Checking now. I never built the site myself so I'll try.

**Comment by lfrancke** [comment]
Site can be built with `mvn site` in the `site` folder.

And it turns out you refer to the picture as `training_logo.svg` (with underscore) but called the file `training-logo.svg` with a dash so it shows up empty.

**Comment by lfrancke** [comment]
And even when fixing the name it's still too large. I'll see if I can fix it but I'm hopefully talentless with CSS

File: site/src/site/site.xml [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
   <bannerLeft>
       <name>Apache Training (incubating)</name>
-      <src>/images/apache_training_logo.png</src>
+      <src>/images/training_logo.svg</src>
```
Comment (lfrancke) [inline site/src/site/site.xml]:
```suggestion
      <src>/images/training-logo.svg</src>
```

**Comment by lfrancke** [comment]
And Chris actually did include instructions on how to build the site here http://training.apache.org/developers/building.html

**Comment by lfrancke** [comment]
I added

```
#bannerLeft > img {
    height: 200px;
}
```
to the CSS file and it looks better in my opinion. Not great, but better.

**Comment by brucearctor** [comment]
@lfrancke  cool, good find:  http://training.apache.org/developers/building.html ... it (mvn site) works.

Hardcoding size in css seems to negate the benefit of SVG (which can scale -- though we do get the transparency).  Though fixed size is likely sufficient.  

I'll play with sizing to make it look even better.  Will close this PR, and start a new/fresh one (I don't know enough, yet, on how to keep modifying this one; open to pointers).

**Comment by lfrancke** [comment]
As I said: I know next to nothing about web design, happy to defer to someone who does.

You can close PRs and start new ones but you can also "reuse" an existing one by just committing more stuff to your existing branch.

So what you'd usually do is fork the project, clone it and then create a branch on your local machine: `git checkout -b TRAINING-16` for example. You'd do your changes, commit them and push them to your fork. `git push` should do the trick usually. Then you create a pull request like you did before and if you just keep on committing to your TRAINING-16 branch all these new changes will automatically appear in the existing pull request

**Pull request closed without merging**